tests/index.test.ts

```diff
@@ -0,0 +1,63 @@
+import * as iots from 'io-ts';
+import * as test from 'tape';
+
+import { reporter } from '../src/';
+
+test('reports an empty array when the result doesn’t contain errors', (t) => {
+    const PrimitiveType = iots.string;
+    const result = PrimitiveType.decode('');
+
+    t.deepEqual(reporter(result), []);
+    t.end();
```

Comment:
[Q]: I'm not familiar with `tape`. Is `t.end` required for synchronous tests? Otherwise we could remove it.

Reply:
It depends. If you are using `t.plan()` to plan the amount of assertions then `t.end()` is not required.
See the documentation [here](https://github.com/substack/tape#testname-opts-cb).

I’m also open for other test runners, personally I use `ava` quite often. But since this project is quite small and doesn’t need things like async testing I thought a lightweight and fast test runner might be enough.

Reply:
I've been working with Jest for 4 years already. So that's all that I know. But I'm fine with new stuff, specially if it's lightweight. We can change it later, it's not a big project.

Reply:
this is already past, but @gillchristian fyi `ava` is near the same as `tape` in the style of its API, doesn't require the `t.end()` call, and [has cleaner output](https://github.com/avajs/ava#why-not-mocha-tape-tap) - so if you'd prefer that I could drop that in in like 3 minutes. although this doesn't and won't do anything async at least you dont need to do that :)

though I recommend against `jest` since it's huge and includes a bunch of stuff that this project doesn't need like automocking and whatever, as well as relying on globals and babel transforms and stuff which makes configuration end up way more complex. `tape` and `ava` are both super simple and do anything this library will probably ever need.

Reply:
I'm fine with AVA as well. Used it when it just came out and I liked it.

@lo1tuma wdyt?

Reply:
AVA is my favourite test runner anyway, so I’m 👍 for this change.